final-react/src/views/Week.jsx

```diff
@@ -33,142 +34,172 @@ function Week() {
   }
 
   const [season, setSeason] = useState(defaultYear);
-  const [week, setWeek] = useState('1');
+  const [week, setWeek] = useState("1");
   const [weekData, setWeekData] = useState([]);
   const [gameData, setGameData] = useState([]);
   const [teamData, dispatchTeamData] = useReducer(teamReducer, []);
   const [seasonData, dispatchSeasonData] = useReducer(seasonReducer, []);
+  const [error, setError] = useState(null);
 
   useEffect(() => {
-    getTeamGameStats(season, week).then((gameData) => setGameData(gameData));
-    getWeekInfo(season, week).then((seasonInfo) => setWeekData(seasonInfo));
+    getTeamGameStats(season, week)
+      .then((gameData) => setGameData(gameData))
+      .catch((error) => {
+        setError("Something went wrong. Please try again!");
+      });
+    getWeekInfo(season, week)
+      .then((seasonInfo) => setWeekData(seasonInfo))
+      .catch((error) => {
+        setError("Something went wrong. Please try again!");
+      });
   }, [season, week]);
 
   useEffect(() => {
-    getTeamInfo().then((teamData) =>
-      dispatchTeamData({ type: 'getTeamData', payload: { teamData: teamData } })
-    );
-    getSeasonStats(season).then((seasonData) =>
-      dispatchSeasonData({
-        type: 'getSeasonData',
-        payload: { seasonData: seasonData },
-      })
-    );
+    getTeamInfo()
+      .then((teamData) =>
+        dispatchTeamData({
+          type: "getTeamData",
+          payload: { teamData: teamData },
+        })
+      )
+      .catch((error) => {
+        setError("Something went wrong. Please try again!");
+      });
+    getSeasonStats(season)
+      .then((seasonData) =>
+        dispatchSeasonData({
+          type: "getSeasonData",
+          payload: { seasonData: seasonData },
+        })
+      )
+      .catch((error) => {
+        setError("Something went wrong. Please try again!");
+      });
   }, []);
 
   const handleWeekChange = (event) => {
     const buttonName = event.target.textContent;
-    setWeek(buttonName.replace(/^\D+/g, ''));
+    setWeek(buttonName.replace(/^\D+/g, ""));
   };
 
   const handleSeasonChange = (event) => {
     const buttonName = event.target.textContent;
-    setSeason(buttonName.replace(/^\D+/g, ''));
+    setSeason(buttonName.replace(/^\D+/g, ""));
+  };
+
+  const handleClose = () => {
+    setError(null);
   };
 
   return (
     <main className="container">
-      <h1>{season}</h1>
-      <h1>Week {week}</h1>
-      <div className="dropdown">
-        <button
-          className="btn btn-secondary dropdown-toggle"
-          type="button"
-          id="dropdownMenuButton"
-          data-bs-toggle="dropdown"
-          aria-haspopup="true"
-          aria-expanded="false"
-        >
-          Choose Season
-        </button>
+      <div className="row mt-5">
```

Comment:
Adding more bootstrap to these makes it look better, thanks for separating them a little bit.